cdap-ui/app/wrangler/components/Wrangler/WrangleData/index.js

```diff
@@ -87,6 +96,89 @@ export default class WrangleData extends Component {
     WranglerStore.dispatch({ type: WranglerActions.redo });
   }
 
+  generateLinks() {
+    let wranglerPluginProperties = this.props.onHydratorApply();
+
+    let batchArtifact = {
```

Comment:
Shouldn't we fetch this from the backend. We have a list endpoint for getting list of artifacts right? This will reduce the hardcoding we do in UI